rtengine/imagedimensions.cc

```diff
@@ -20,49 +20,88 @@
 #include "imagedimensions.h"
 #include "rtengine.h"
 
-void ImageDimensions::transform (PreviewProps pp, int tran, int &sx1, int &sy1, int &sx2, int &sy2)
+PreviewProps::PreviewProps(int _x, int _y, int _width, int _height, int _skip) :
```

Comment:
You do not need to add underscore for local variables, `x(x)` is a valid syntax and already used several times in RT.